src/main/kotlin/org/malachite/estella/services/OrganizationService.kt

```diff
@@ -30,7 +30,10 @@ class OrganizationService(
     }
 
     fun getOrganizationByUser(user: User): Organization =
-        organizationRepository.findByUser(user).get()
+        (user.id
+            ?.let { organizationRepository.findByUserId(it) }
```

Comment:
W organizacji mamy już taką metodę? Tam nie było tak, że szukaliśmy właśnie po całym userze, nie tylko jego ID?

Reply:
Jest byUserId

Reply:
@Davenury jeśli chodzi ci o to czy nie mamy po cały Userze i tu jej użyć to już takiej nie mamy bo stwierdziliśmy że będziemy się trzymać po id chyba XD